history-stream.js

```diff
@@ -16,17 +16,25 @@ exports.init = function (sbot, config) {
   sbot.createHistoryStream = function(opts) {
     // default values
     const seq = opts.sequence || opts.seq || 0
-    const limit = opts.limit || 1e10
+    const limit = opts.limit
     const keys = opts.keys === false ? false : true
     const values = opts.values === false ? false : true
 
-    const query = {
-      type: 'EQUAL',
-      data: {
-        seek: sbot.db.jitdb.seekAuthor,
-        value: opts.id,
-        indexType: "author",
-        indexAll: true
+    let query = require("./operators").author(opts.id)
```

Comment:
Question :
 Are you making something with exactly same api? Seems like if it's going to be subdomained under ssb.db, changing the api there might be fine? 

 given id isn't an option, I would prefer `createHistoryStream(id, opts)`

Either way a check that id is a feedId could be important? I'm asking because intetested to know how and where others are checking inputs like this

Reply:
Yeah it is meant to have the exactly same API as it would be used for legacy replication. I don't think it makes sense the implement the other streaming methods. I added a check + test for the feed thing, thanks :-)